ingestion/src/metadata/test_suite/validations/column/column_value_max_to_be_between.py

```diff
@@ -46,7 +47,9 @@ def column_value_max_to_be_between(
     """
 
     try:
-        column_name = test_case.entityLink.__root__.split("::")[-1].replace(">", "")
+        column_name = unquote_plus(
+            test_case.entityLink.__root__.split("::")[-1].replace(">", "")
+        )
```

Comment:
Hey @TeddyCr, PR looks good just one suggestion, can we write this logic in a separate function and use that function everywhere, so if there are some changes required in the future for this logic we just need to update the function.